### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by invictusro** [scivision/PyLivestream#92] [Q&A]
Hi, is there any way I can stream to another service than the ones presented if I have stream key and stream URL ? 
Basically to stream to a RTMP server, but not for the ones present there ?

**Comment by tabanger** [comment]
Yes, you can stream to any RTMP server; just add your own server configuration in the pylivestream.ini file.  Use the other sites as an example.  For example, you could add this section to the ini file:

[myserver]
server: rtmp://your.server.address
key: ~/yourkey.key

**Comment by scivision** [comment]
In PyLivestream >= 2.0, the new JSON pylivestream.json (or whatever name you want the file to be) can add any arbitrary server. See the example [pylivestream.json](https://github.com/scivision/PyLivestream/blob/main/src/pylivestream/data/pylivestream.json), copy it to another file and edit it to add your streaming service. It needs at least the fields "url" and "streamid".
